shared/config.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable canonical/filename-match-exported */
+const contractName = process.env.CONTRACT_NAME ?? 'near-transactions-accounting-report';
+const localHome = process.env.HOME;
+
+// eslint-disable-next-line max-lines-per-function
+const getConfig = (environment: string = 'development') => {
+  console.log('shared/config.js getConfig', { environment });
+  switch (environment) {
+    case 'production':
+    case 'mainnet':
+      return {
+        contractName,
+        exampleAccount: 'example.near',
+        explorerUrl: 'https://explorer.mainnet.near.org',
+        helperUrl: 'https://helper.mainnet.near.org',
+        networkId: 'mainnet',
+        nodeUrl: 'https://rpc.mainnet.near.org',
+        walletUrl: 'https://wallet.near.org',
```

Comment:
Note for the future, you will have to change it to mynearwallet or ideally to the wallet selector.